ProjFS.Mac/PrjFSKext/PrjFSKext/KauthHandler.cpp

```diff
@@ -610,16 +608,23 @@ static bool ShouldHandleVnodeOpEvent(
         }
     }
     
-    *root = VirtualizationRoots_FindForVnode(vnode);
+    *rootIndex = VirtualizationRoot_FindForVnode(vnode);
+    
+    // This VNode is in the temp directory, do not act on it.
```

Comment:
Is this leftover from when you were looking into the atomic placeholder/temp directory work?

Reply:
Yeah, I was unsure whether to remove it - it's also in your atomic enumeration branch. I definitely wanted to pull in the enum with the "special" root indices and finally get rid of the -1s everywhere. It seemed silly to remove the temp directory enum value, and we're presumably going to need this condition here, even if it's not going to kick in in this version of the code. I can remove it though.